controllers/src/main/scala/cromwell/pipeline/controller/ProjectFileController.scala

```diff
@@ -38,7 +39,10 @@ class ProjectFileController(wdlService: ProjectFileService)(
                 (validateResponse, uploadResponse) match {
                   case (Right(_), Right(response)) => StatusCodes.OK.intValue -> response
                   case (Left(_), Right(response))  => StatusCodes.Created.intValue -> response
-                  case (_, Left(response))         => StatusCodes.UnprocessableEntity.intValue -> response.message
+                  case (_, Left(response))         => response match {
+                    case HttpException(message) => StatusCodes.UnprocessableEntity.intValue -> message
```

Comment:
Not sure if this has to be handled like this

Reply:
I believe here we can use exception types in patterns to avoid inner pattern matching:
```case (_, Left(HttpException(message))) =>  StatusCodes.UnprocessableEntity.intValue -> message 
case (_, Left(response)) => StatusCodes.UnprocessableEntity.intValue -> "Bad request"```

Reply:
Thanks!!! I was not sure about include all the exceptions, it makes sense to handle the exception as you proposed

Reply:
Why not? They are handled in the same way.
If we decide to handle some of them differently, we can split them again.